### ISSUE TITLE
[Feature Request] Buttons Labels should be customizable with result of a command

### ISSUE THREAD

**Issue by martindoublem** [phuhl/linux_notification_center#42]
Hi there,

I love your notification center, it was clearly something missing for a light WM.

My idea is that for buttons label you should be able to return the result of a command. Such as something that would permit: Wifi + SSID + IP, or bluetooth + number of device connected. Does this make sense?

**Comment by phuhl** [comment]
Yeah, I thought about something like that, too. Maybe in the next release, am quite busy right know. Anyways, thank you for the feedback!
